rust/main/agents/scraper/migration/bin/init_db.rs

```diff
@@ -4,7 +4,7 @@ use common::*;
 
 mod common;
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
 async fn main() -> Result<(), DbErr> {
```

Comment:
I wonder if it produces the same database schema as before (i.e. old version vs new version)?

Reply:
It appears so, I did the test which you mentioned in the issue of the description, which is to generate a schema on the old version, scrape a bit, then switch to the new version and continue scraping, everything worked fine

Reply:
It is good that it works. It is not the test for this comment though (but a good test for [that comment](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4925#issuecomment-2517970903))

The test is mostly manual for this:
1. Start Postgres on one port.
2. Init schema there (`init_db.rs`) from `main`.
3. Start Postgres on a different port.
4. Init schema from this branch.
5. Compare the tables and field types.

Reply:
Great! So, database schema is the same.